Packs/AzureWAF/Integrations/AzureWAF/AzureWAF_description.md

```diff
@@ -1,14 +1,36 @@
-In both options below, the [device authorization grant flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-device-code) is used.
+# Self-Deployed Application
+To use a self-configured Azure application, you need to add a [new Azure App Registration in the Azure Portal](https://docs.microsoft.com/en-us/graph/auth-register-app-v2#register-a-new-application-using-the-azure-portal).
 
-In order to connect to Azure Web Application Firewall using either the Cortex XSOAR Azure or Self Deployed Azure application:
-1. Fill in the required parameters
-2. Run the ***!azure-waf-auth-start*** command.
-3. Follow the instructions that appear.
-4. Run the ***!azure-waf-auth-complete*** command.
-At end of the process, you will see a message that you logged in successfully. 
+## Required Permissions:
+1. user_impersonation
+2. offline_access
+3. user.read 
+
+## Authentication Using the  User-Authentication Flow (recommended)
+
+Follow these steps for a self-deployed configuration:
+
+1. To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. To add the registration, refer to the following [Microsoft article](https://docs.microsoft.com/en-us/microsoft-365/security/defender/api-create-app-web?view=o365-worldwide#create-an-app) steps 1-8.
```

Comment:
isnt this a duplicate of a few lines above?

Reply:
I don't think so